ydb/library/yql/providers/dq/runtime/file_cache.cpp

```diff
@@ -123,7 +123,7 @@ void TFileCache::AddFile(const TString& path, const TString& objectId)
         LRU.push_back(objectId);
         file.Position = --LRU.end();
         UsedSize += file.Size;
-        Files.insert(std::make_pair(objectId, file));
+        Files[objectId] = file;
```

Comment:
if objectId was previously present, UsedSize must be reduced by the previous file size.